google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/apis/GoogleCloudApiManagementPanel.java

```diff
@@ -0,0 +1,278 @@
+    selectedApisTable
+        .getColumnModel()
+        .getColumn(CLOUD_API_NAME_COL)
+        .setMinWidth(MIN_CLOUD_API_COL_WIDTH);
+    selectedApisTable
+        .getColumnModel()
+        .getColumn(CLOUD_API_ENABLEMENT_COL)
+        .setMinWidth(MIN_CLOUD_API_ENABLEMENT_COL_WIDTH);
+
+    allApisTable.setModel(
+        new ApisTableModel(Sets.difference(Sets.newHashSet(allLibraries), selectedLibraries)));
+    allApisTable
+        .getColumnModel()
+        .getColumn(CLOUD_API_NAME_COL)
+        .setMinWidth(MIN_CLOUD_API_COL_WIDTH);
+    allApisTable
+        .getColumnModel()
+        .getColumn(CLOUD_API_ENABLEMENT_COL)
+        .setMinWidth(MIN_CLOUD_API_ENABLEMENT_COL_WIDTH);
+  }
+
+  JPanel getPanel() {
+    return panel;
+  }
+
+  @VisibleForTesting
+  JBScrollPane getApisScrollPane() {
+    return apisScrollPane;
+  }
+
+  @VisibleForTesting
+  JLabel getSelectProjectLabel() {
+    return selectProjectLabel;
+  }
+
+  @VisibleForTesting
+  JTable getSelectedApisTable() {
+    return selectedApisTable;
+  }
+
+  @VisibleForTesting
+  JTable getAllApisTable() {
+    return allApisTable;
+  }
+
+  @VisibleForTesting
+  ProjectSelector getProjectSelector() {
+    return projectSelector;
+  }
+
+  private void createUIComponents() {
+    selectedApisTable = new ApisTable();
+    selectedApisTable.setTableHeader(null);
+
+    allApisTable = new ApisTable();
+    allApisTable.setTableHeader(null);
+  }
+
+  private static final class ApisTable extends JBTable {
+    ApisTable() {
+      setDefaultRenderer(CloudLibrary.class, new CloudLibraryRenderer());
+      setDefaultRenderer(Boolean.class, new CloudApiEnablementRenderer());
+    }
+  }
+
+  /** Custom {@link TableModel} for the API enablement table instances. */
+  @VisibleForTesting
+  static final class ApisTableModel implements TableModel {
+
+    private final SortedMap<CloudLibrary, Boolean> librariesToEnabledStatus =
+        new TreeMap<>(Comparator.comparing(CloudLibrary::getName));
+
+    ApisTableModel(Collection<CloudLibrary> libraries) {
+      librariesToEnabledStatus.putAll(Maps.toMap(libraries, this::getEnabledStatus));
+    }
+
+    boolean getEnabledStatus(CloudLibrary library) {
+      // TODO(eshaul) query the IAM API to get the actual status of the API
+      return false;
+    }
+
+    @Override
+    public int getRowCount() {
+      return librariesToEnabledStatus.size();
+    }
+
+    @Override
+    public int getColumnCount() {
+      return 2;
+    }
+
+    @Override
+    public String getColumnName(int columnIndex) {
+      return null;
+    }
+
+    @Override
+    public Class<?> getColumnClass(int columnIndex) {
+      if (columnIndex == 0) {
+        return CloudLibrary.class;
+      }
+
+      return Boolean.class;
+    }
+
+    @Override
+    public boolean isCellEditable(int rowIndex, int columnIndex) {
+      return false;
+    }
+
+    @Override
+    public Object getValueAt(int rowIndex, int columnIndex) {
+      return columnIndex == 0
```

Comment:
How about using `switch` here? It's pretty clear from `switch` how many columns we have and how we assign them values whereas here it looks like we only provide separate value for column 0 here. And also we might use constants instead of number (i.e. `CLOUD_API_NAME_COL`) maybe? Same idea about `getColumnClass`